articles/virtual-machines/linux/maintenance-in-place.md

```diff
@@ -44,8 +44,10 @@ to virtual machines.
 Multi-instance updates (VMs in an availability set) are applied one
 update domain at a time.
 
+Some applications may be impacted by these updates more than others. 
+Applications that perform real-time event processing, media streaming 
+or transcoding, or high throughput networking scenarios, for example,
+may not be designed to tolerate a 30 second pause. 
 Applications running in a virtual machine can learn about upcoming
-updates by calling the metadata service scheduled events. For more
-information about scheduled events, refer to [Azure Metadata Service -
-Scheduled
-Events](../virtual-machines-scheduled-events.md).
\ No newline at end of file
+updates by calling the [Scheduled Events](../virtual-machines-scheduled-events.md)
+API of the [Azure Metadata Service](../virtual-machines-instancemetadataservice-overview.md).
```

Comment:
Not sure I follow, Hari. Both the Metadata Service and Schedule Events docs are linked here. Are there additional docs that should be linked here?